scss/_variables.scss

```diff
@@ -39,10 +39,10 @@ $o-buttons-sizes: (
 		border-width: 1px,
 	),
 	medium: (
-		font-size: 16px,
-		height: 24px,
-		padding-left: 14px,
-		padding-right: 14px,
+		font-size: 14px,
```

Comment:
Pretty sure this needs to stay at 16px to match default text size of body and form fields.